src/tests/ftest/rebuild/cascading_failures.yaml

```diff
@@ -8,6 +8,9 @@ server_config:
     targets: 2
     nr_xs_helpers: 0
     scm_size: 4
+    log_mask: DEBUG
+    env_vars:
+      - DD_MASK=all
```

Comment:
Just checking - do we intend to keep this level of debug turned on?

Reply:
This is only for debugging purpose for this patch, since I can not reproduce the failure locally, so I need collect more logs here. I will remove this once fixing the CI issue.